### DISCUSSION TITLE
Cheat sheet or quick way to determine if i should use static generation or server side render?

### DISCUSSION THREAD

**Discussion by mikkodimapasoc** [vercel/next.js#17432] [Help] [answered]
Hi!
Newbie here. Just finished the learn section of the docs. To be honest im still kinda confused in deciding on which approach should i use when rendering my pages. Technically it would run on either approach. I want to stick to the best practice. Is there a quick way to determine if i should statically generate or server side render my page? How would i know if one is better than the other? Are there things that i should always keep in mind?

**Comment by ErnestBrandi** [comment, marked as answer]
Hey, maybe you should read that article :) 
https://vercel.com/blog/nextjs-server-side-rendering-vs-static-generation

**Reply by mikkodimapasoc** [reply]
Thank you!
